tools/cpp/osx_cc_configure.bzl

```diff
@@ -120,30 +118,17 @@ def configure_osx_toolchain(repository_ctx, overriden_tools):
         if (xcrun_result.return_code == 0):
             repository_ctx.symlink("wrapped_clang", "wrapped_clang_pp")
         else:
-            # If for some reason wrapped_clang couldn't be built, fall back to
-            # using the bash scripts that don't support dSYM generation. This is to
-            # avoid completely breaking a build. This should be removed after a whole
-            # release cycle to keep from increasing code maintenance, if we haven't
-            # received any issues as requested below.
             error_msg = (
                 "return code {code}, stderr: {err}, stdout: {out}"
             ).format(
                 code = xcrun_result.return_code,
                 err = xcrun_result.stderr,
                 out = xcrun_result.stdout,
             )
-            print("wrapped_clang failed to generate. This shouldn't cause " +
-                  "problems, but please file an issue at " +
-                  "https://github.com/bazelbuild/bazel/issues with the following:\n" +
-                  error_msg)
-            repository_ctx.symlink(
-                paths["@bazel_tools//tools/osx/crosstool:wrapped_clang.tpl"],
-                "wrapped_clang",
-            )
-            repository_ctx.symlink(
-                paths["@bazel_tools//tools/osx/crosstool:wrapped_clang_pp.tpl"],
-                "wrapped_clang_pp",
-            )
+            fail("wrapped_clang failed to generate. This shouldn't cause " +
```

Comment:
Remove the "this shouldn't cause problems" part, since it will cause a problem now :)

Reply:
Ha yea done, left the part about filing an issue